StoryFactory.gd

```diff
@@ -0,0 +1,47 @@
+extends Node
+
+class_name StoryFactory
```

Comment:
It probably makes sense for the StoryFactory to be a singleton but idk how to do it and I don't think it's strictly necessary right now.